File: fortnite/templates/fortnite_template.cpp
#include <iostream>

using namespace std;

/**
 * Return the shortest time for you to escape the storm.
 *
 * N: starting health
 * H: healing per second
 * D: distance out of the storm in meters
 * S: running speed in meters per second
 * P: storm damage per second
 */
long long solve(long long N, long long H, long long D, long long S, long long P) {
    // YOUR CODE HERE
    return -1;
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        long long N, H, D, S, P;
        cin >> N >> H >> D >> S >> P;
        cout << solve(N, H, D, S, P) << '\n';
    }
}

File: fortnite/templates/fortnite_template.py
def solve(N: int, H: int, D: int, S: int, P: int, T: int) -> int:
    """
    Return the shape of displayed by ASCII string G of dimensions M x N
    
    N: starting health
    H: amount of healing
    D: distance out of the storm in meters
    S: running speed in meters per second
    P: storm damage per second
    T: time to heal
    """
    # YOUR CODE HERE
    return -1

def main():
    T = int(input())
    for _ in range(T):
        N, H, D, S, P, T = map(int, input().split())
        print(solve(N, H, D, S, P, T))

if __name__ == '__main__':
    main()